<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="4" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-07-10T17:12:25 IST" name="Suite" finished-at="2025-07-10T17:13:19 IST" duration-ms="53392">
    <groups>
    </groups>
    <test started-at="2025-07-10T17:12:25 IST" name="Test" finished-at="2025-07-10T17:13:19 IST" duration-ms="53392">
      <class name="com.Testcases.TC_scroll_Click_Switch_004">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.Testcases.TC_scroll_Click_Switch_004@5e600dd5]" started-at="2025-07-10T17:13:00 IST" name="setup" finished-at="2025-07-10T17:13:05 IST" duration-ms="5416" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="switchScroll()[pri:0, instance:com.Testcases.TC_scroll_Click_Switch_004@5e600dd5]" started-at="2025-07-10T17:13:05 IST" name="switchScroll" finished-at="2025-07-10T17:13:19 IST" duration-ms="13370" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- switchScroll -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:com.Testcases.TC_scroll_Click_Switch_004@5e600dd5]" started-at="2025-07-10T17:13:19 IST" name="TearDown" finished-at="2025-07-10T17:13:19 IST" duration-ms="0" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
sw cannot be resolved to a type
Syntax error on token ".", ; expected
The method quit() is undefined for the type baseclass
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
sw cannot be resolved to a type
Syntax error on token ".", ; expected
The method quit() is undefined for the type baseclass

at com.base.baseclass.TearDown(baseclass.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:569)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.Testcases.TC_scroll_Click_Switch_004 -->
      <class name="com.Testcases.TC_Notification_003">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.Testcases.TC_Notification_003@130c12b7]" started-at="2025-07-10T17:12:49 IST" name="setup" finished-at="2025-07-10T17:12:54 IST" duration-ms="4425" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Notify()[pri:0, instance:com.Testcases.TC_Notification_003@130c12b7]" started-at="2025-07-10T17:12:54 IST" name="Notify" finished-at="2025-07-10T17:13:00 IST" duration-ms="6153" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Notify -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:com.Testcases.TC_Notification_003@130c12b7]" started-at="2025-07-10T17:13:00 IST" name="TearDown" finished-at="2025-07-10T17:13:00 IST" duration-ms="0" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
sw cannot be resolved to a type
Syntax error on token ".", ; expected
The method quit() is undefined for the type baseclass
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
sw cannot be resolved to a type
Syntax error on token ".", ; expected
The method quit() is undefined for the type baseclass

at com.base.baseclass.TearDown(baseclass.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:569)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.Testcases.TC_Notification_003 -->
      <class name="com.Testcases.TC_Alert_001">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.Testcases.TC_Alert_001@242b836]" started-at="2025-07-10T17:12:26 IST" name="setup" finished-at="2025-07-10T17:12:32 IST" duration-ms="5692" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Alert()[pri:0, instance:com.Testcases.TC_Alert_001@242b836]" started-at="2025-07-10T17:12:32 IST" name="Alert" finished-at="2025-07-10T17:12:38 IST" duration-ms="6119" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Alert -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:com.Testcases.TC_Alert_001@242b836]" started-at="2025-07-10T17:12:38 IST" name="TearDown" finished-at="2025-07-10T17:12:38 IST" duration-ms="0" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
sw cannot be resolved to a type
Syntax error on token ".", ; expected
The method quit() is undefined for the type baseclass
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
sw cannot be resolved to a type
Syntax error on token ".", ; expected
The method quit() is undefined for the type baseclass

at com.base.baseclass.TearDown(baseclass.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:569)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.Testcases.TC_Alert_001 -->
      <class name="com.Testcases.TC_Animation_Reversing_002">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.Testcases.TC_Animation_Reversing_002@121314f7]" started-at="2025-07-10T17:12:38 IST" name="setup" finished-at="2025-07-10T17:12:42 IST" duration-ms="4334" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="reversing()[pri:0, instance:com.Testcases.TC_Animation_Reversing_002@121314f7]" started-at="2025-07-10T17:12:42 IST" name="reversing" finished-at="2025-07-10T17:12:49 IST" duration-ms="7376" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reversing -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:com.Testcases.TC_Animation_Reversing_002@121314f7]" started-at="2025-07-10T17:12:49 IST" name="TearDown" finished-at="2025-07-10T17:12:49 IST" duration-ms="0" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
sw cannot be resolved to a type
Syntax error on token ".", ; expected
The method quit() is undefined for the type baseclass
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
sw cannot be resolved to a type
Syntax error on token ".", ; expected
The method quit() is undefined for the type baseclass

at com.base.baseclass.TearDown(baseclass.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:569)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.Testcases.TC_Animation_Reversing_002 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
